app/common/helpers.py

```diff
@@ -0,0 +1,97 @@
+from flask_restful import abort
+
+from app.models import User, BucketList, BucketListItem
+from app import db
+
+
+def register_user(user):
+    """
+    After a successfull registration,
+    it welcomes the user.
+    """
+    return {
+        'message': 'Welcome ' + str(user.username) + ' to the Bucketlist Service.'
```

Comment:
**Crucial(LO)**
Adding string using the `+` sign is inefficient. Why? because strings are immutable in python and as such when you use the plus sign you are in effect creating new strings. And although this might not be visible in the case of three strings, it is good to get into the habit of not doing it that way.

In python, there are various ways to do string formatting but for consistency I always suggest the use of `.format()` method.